CONTRIBUTING.rst

```diff
@@ -96,19 +116,21 @@ In order to update the version and change log entry, you have to create a
 release file. This is a normal restructured text file called RELEASE.rst that
 lives in the root of the repository and will be used as the change log entry.
 
-It should start with following lines:
-
-* RELEASE_TYPE: major
-* RELEASE_TYPE: minor
-* RELEASE_TYPE: patch
-
-This specifies the component of the version number that should be updated, with
-the meaning of each component following `semver <http://semver.org/>`_. As a
-rule of thumb if it's a bug fix it's probably a patch version update, if it's
-a new feature it's definitely a minor version update, and you probably
-shouldn't ever need to use a major version update unless you're part of the
-core team and we've discussed it a lot.
-
+The first line of the file specifies the component of the version number that
+will be updated, according to our `semantic versioning <http://semver.org/>`_
+policy.
```

Comment:
Having this in the `CONTRIBUTING.rst` seems weird. It should probably go in a guide somewhere (not sure where) and linked to from here.